include/ROSEndEffector/FindActions.h

```diff
@@ -0,0 +1,77 @@
+#ifndef __ROSEE_FIND_ACTIONS_
+#define __ROSEE_FIND_ACTIONS_
+
+#include <moveit/planning_scene/planning_scene.h>
+
+#include <ROSEndEffector/YamlWorker.h>
+#include <ROSEndEffector/ParserMoveIt.h>
+#include <ROSEndEffector/ActionPinchStrong.h>
+#include <ROSEndEffector/ActionPinchWeak.h>
+#include <ROSEndEffector/ActionTrig.h>
+
+
+#define N_EXP_COLLISION 5000 //5000 is ok
+#define N_EXP_DISTANCES 5000 //? is ok
+#define DEFAULT_JOINT_POS 0.0
+
+namespace ROSEE
+{
+    
+/**
+ * @brief Class to check which fingertips collide (for the pinch action at the moment)
+ * 
+ * @warning there is a problem with collisions: with the schunk hand, if we only the middle (base phalange)
+ * toward the hand, a collision between index tip, middle tip and ring tip is detected. Easy reproducible with the 
+ * moveit assistant, in the set pose section (it find a collision when visually is not present, when we move the 
+ * middle). There are some caotic printing in bugmoveit branch, to replicate the problem also with this code.
+ * I dont know if it is a problem of schunk model, moveit, or both.
```

Comment:
I see, probably we should not worry about this at this stage: let's check other hands model in the future